### ISSUE TITLE
subchart as plugin

### ISSUE THREAD

**Issue by c-arnab** [TargetProcess/tauCharts#367]
Would love the subchart plugin as in [http://c3js.org/samples/options_subchart.html](http://c3js.org/samples/options_subchart.html).

A use case would be the subchart shows data in days and when a area is selected the main chart x axis shows data upto seconds scale

Thanks
Arnab

**Comment by vladminsky** [comment]
@c-arnab 

Yes, it is possible to bind charts using plugins, like in the following example from taucharts blog (https://blog.taucharts.com/viz-as-query-language/):

https://jsfiddle.net/taucharts/rLgdqux0/

But we still have no generic "Brush" plugin to build exact subcharts you ask above.
I will try to prototype some proof-of-concept this week. Several peoples ask this feature for a long time...
